generator/tools/templates/react/README.md

```diff
@@ -0,0 +1,18 @@
+# applet-template
+
+## Dev Instructions
+
+-   clone directly into `[AppletSys]/ui` directory
+-   update contract name at bottom of `vite.config.ts` (`plugins: [react(), psibase("token-sys")],`)
+-   `yarn` to install dependencies
+-   `yarn dev` and open on port `8081` for HMR!
+
+## Build Instructions
+
+-   building will create `dist` directory
+-   `psibase upload-tree [contract-name] / ./dist`
+
+## Making sure your built files are bootstrapped with new tester and new chain
```

Comment:
Sounds like maybe this README _isn't_ targeting third-party devs. If it is, most probably also wouldn't need this section.